testcases/misc_tests/p11sak_test.sh

```diff
@@ -517,6 +541,12 @@ echo "* TESTCASE list-key ec PASS Listed random ec public keys CK_BYTE attribute
 else
 echo "* TESTCASE list-key ec FAIL Failed to list ec public keys CK_BYTE attribute"
 fi
+# URI
+if [[ $(grep -A 85 'p11sak-ec-prime256v1:pub' $P11SAK_EC_LONG | grep -c 'URI: pkcs11:.*type=public') == "1" ]]; then
+echo "* TESTCASE list-key ec PASS list ec public key pkcs#11 URI"
+else
+echo "* TESTCASE list-key ec FAIL list ec public key pkcs#11 URI"
+fi
 
 
```

Comment:
Is there a reason why no private keys are listed? 
No need to add them now for this PR if they did not exist jet, just wondering....

Reply:
One reason might be, that the p11sak listings lists multiple keys, but only the first one in the listing is selected for the test. It is hard in shell scripts to separate multiple multi-line objects from a logfile. The easy way is to pick only the first one. But this is only my guess.

Choosing perl or python for these testscripts would make that job much easier.